syncpack.config.cjs

```diff
@@ -183,6 +188,18 @@ module.exports = {
 			packages: ["**"],
 			isIgnored: true,
 		},
+
+		// Prettier is ignored while the eslint-config-fluid is in the client release group. This can be removed once
+		// eslint-config-fluid is back on its own, or we otherwise unify on a single prettier version.
+		{
+			label:
+				"Allow multiple versions of prettier while eslint-config-fluid is in the client release group",
+			dependencies: ["prettier"],
+			dependencyTypes: ["dev", "prod"],
+			packages: ["**"],
+			isIgnored: true,
+		},
+
```

Comment:
The PR seems to have unified on prettier 3.6.2 so this should be unnecessary now, right?